pkg/proxy/iptables/proxier.go

```diff
@@ -538,7 +547,8 @@ func (proxier *Proxier) syncProxyRules() {
 				} else {
 					socket, err := openLocalPort(&lp)
 					if err != nil {
-						glog.Errorf("can't open %s, skipping this externalIP: %v", lp.String(), err)
+						proxier.recorder.Eventf(proxier.nodeRef, "FailedToOpenExternalIP", "Can't open %s, skipping this externalIP", lp.String())
```

Comment:
reading the PR top-to-bottom: all the previous events are about something actually going wrong on the node.  This event is a) about the Service, so it should have a ref to the service, not the node and b) user-controlled so we should be careful not to allow DoS.  More on this when I wrap up the review.